libs/langchain-mcp-adapters/src/tools.ts

```diff
@@ -237,7 +239,7 @@ export async function loadMcpTools(
   serverName: string,
   client: Client,
   options?: LoadMcpToolsOptions
-): Promise<StructuredToolInterface[]> {
+): Promise<(StructuredToolInterface | ToolInterface)[]> {
```

Comment:
Is `ToolInterface` not assignable to `StructuredToolInterface`? I thought we fixed that recently, apologies if not.